var/spack/repos/builtin/packages/mpark-variant/package.py

```diff
@@ -19,5 +19,14 @@ class MparkVariant(CMakePackage):
     # Ref.: https://github.com/mpark/variant/pull/73
     patch('nvcc.patch', when='@:1.4.0')
 
-    conflicts('%gcc@:4.7')
-    conflicts('%clang@:3.5')
+    cxx11_msg = 'MPark.Variant needs at a C++11-capable compiler. ' \
+                'See https://github.com/mpark/variant#requirements'
+    conflicts('%gcc@:4.7', msg=cxx11_msg)
+    conflicts('%clang@:3.5', msg=cxx11_msg)
+
+    conflicts('%gcc@7.3.1',
+              msg='GCC 7.3.1 has a bug that prevents using MPark.Variant. '
+                  'See https://github.com/mpark/variant/issues/43 and '
+                  '    https://gcc.gnu.org/bugzilla/show_bug.cgi?id=84785 '
```

Comment:
```suggestion
                  'https://gcc.gnu.org/bugzilla/show_bug.cgi?id=84785 '
```
You don't have any new-line characters here, so it won't be wrapped until it hits the edge of the terminal